config/jobs/kubernetes/kops/build_jobs.py

```diff
@@ -925,6 +925,23 @@ def generate_misc():
                    runs_per_day=3),
 
 
+        # [sig-network, @aojea] A job to test SIG Network beta features on Kops to validate "real-world" usage availability.
```

Comment:
I will like if I can receive also alerts from this job, so I can be notified on failures

Reply:
Not sure how to do that TBH - does prow support this?

Reply:
https://github.com/kubernetes/test-infra/blob/7ba5dbe703fbd4bdf3c693a7e24972634703c215/config/jobs/kubernetes-sigs/kind/kind.yaml#L77